package.json

```
@@ -30,57 +28,54 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~12.1.4",
-    "@angular/common": "~12.1.4",
-    "@angular/compiler": "~12.1.4",
-    "@angular/core": "~12.1.4",
-    "@angular/forms": "~12.1.4",
-    "@angular/platform-browser": "~12.1.4",
-    "@angular/platform-browser-dynamic": "~12.1.4",
-    "@angular/router": "~12.1.4",
-    "rxjs": "~6.6.3",
+    "@angular/animations": "~13.1.0",
+    "@angular/common": "~13.1.0",
+    "@angular/compiler": "~13.1.0",
+    "@angular/core": "~13.1.0",
+    "@angular/forms": "~13.1.0",
+    "@angular/platform-browser": "~13.1.0",
+    "@angular/platform-browser-dynamic": "~13.1.0",
+    "@angular/router": "~13.1.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jsdoc": "^37.2.0",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
+    "rxjs": "~7.4.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~12.1.4",
-    "@angular/cli": "~12.1.4",
-    "@angular/compiler-cli": "~12.1.4",
-    "@angular/language-service": "~12.1.4",
-    "@swimlane/prettier-config-swimlane": "^3.0.2",
-    "@types/jasmine": "~3.6.0",
+    "@angular-devkit/build-angular": "~13.1.1",
+    "@angular-eslint/builder": "13.0.1",
+    "@angular-eslint/eslint-plugin": "13.0.1",
+    "@angular-eslint/eslint-plugin-template": "13.0.1",
+    "@angular-eslint/schematics": "13.0.1",
+    "@angular-eslint/template-parser": "13.0.1",
+    "@angular/cli": "~13.1.1",
+    "@angular/compiler-cli": "~13.1.0",
+    "@angular/language-service": "~13.1.0",
+    "@types/jasmine": "~3.10.0",
     "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^12.11.1",
+    "@types/node": "~16.11.12",
+    "@typescript-eslint/eslint-plugin": "5.3.0",
+    "@typescript-eslint/parser": "5.3.0",
     "angular-cli-ghpages": "^0.6.0",
-    "codelyzer": "^6.0.0",
     "cpx": "^1.5.0",
     "cross-env": "^7.0.2",
-    "husky": "^4.2.4",
+    "eslint": "^8.2.0",
     "jasmine": "^3.5.0",
-    "jasmine-core": "~3.6.0",
-    "jasmine-spec-reporter": "~5.0.0",
+    "jasmine-core": "~3.10.1",
+    "jasmine-spec-reporter": "~7.0.0",
     "karma": "~6.3.4",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "ng-packagr": "^12.1.2",
+    "ng-packagr": "^13.1.1",
     "node-sass": "^7.0.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.0.4",
-    "pretty-quick": "^2.0.1",
     "protractor": "~7.0.0",
     "scss-bundle": "^3.1.1",
-    "ts-node": "~7.0.0",
-    "tslint": "~6.1.0",
-    "tslint-config-prettier": "^1.17.0",
-    "tslint-config-security": "^1.16.0",
-    "tslint-config-swimlane": "^5.0.0",
-    "typescript": "^4.3.5"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "pretty-quick --staged"
-    }
+    "ts-node": "~9.1.1",
+    "typescript": "^4.4.3"
```

```suggestion
    "typescript": "~4.4.3"
```

because TS 4.5 is still picked otherwise